### PR TITLE
fix(service-account): create add-cluster-modal & update PIconModal spec

### PR DESCRIPTION
**Pull request by piggggggggy**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci`, minor refactoring, etc.)
- [ ] Need discussion
- [ ] Not that difficult
- [ ] Approved feature branch merge to master

### Description
SSIA

image

### CODE REVIEW

**Review by yuda110 — APPROVED** [review]
LGTM!!!!

File: apps/web/src/services/asset-inventory/components/ServiceAccountAddClusterModal.vue [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+                                {{ $t('INVENTORY.SERVICE_ACCOUNT.CLUSTER_MODAL.INFORMATION') }}
+                            </span>
+                        </div>
+                    </div>
+                </div>
+                <div class="button-wrapper">
+                    <p-button class="continue-button"
+                              icon-right="ic_arrow-right"
+                    >
+                        Continue
```
Comment (sulmoJ) [inline apps/web/src/services/asset-inventory/components/ServiceAccountAddClusterModal.vue:80]: nitpick: translate!

File: apps/web/src/services/asset-inventory/components/ServiceAccountAddClusterModal.vue [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+                    </p-button>
+                    <p-button style-type="transparent">
+                        Not now
```
Comment (sulmoJ) [inline apps/web/src/services/asset-inventory/components/ServiceAccountAddClusterModal.vue:83]: nitpick: translate!

**Review by sulmoJ — APPROVED** [review]
LGTM~

**Pull request merged**